src/GovUk.Education.ExploreEducationStatistics.Admin/Controllers/Api/MetaController .cs

```diff
@@ -0,0 +1,29 @@
+using System.Collections.Generic;
+using GovUk.Education.ExploreEducationStatistics.Admin.Models;
+using GovUk.Education.ExploreEducationStatistics.Admin.Services.Interfaces;
+using Microsoft.AspNetCore.Authorization;
+using Microsoft.AspNetCore.Mvc;
+
+namespace GovUk.Education.ExploreEducationStatistics.Admin.Controllers.Api
+{
+    [ApiController]
+    [Authorize]
+    [Route("/api")]
+    public class MetaController : ControllerBase
+    {
+        private readonly IMetaService _metaService;
+
+        public MetaController(IMetaService metaService)
+        {
+            _metaService = metaService;
+        }
+
+        // GET api/meta/timeidentifiers
+        [HttpGet("/meta/timeidentifiers")]
```

Comment:
can perhaps put `/met`a at the controller level?

Reply:
will do

Reply:
Done